FileLiberator/DownloadDecryptBook.cs

```diff
@@ -214,11 +201,11 @@ var title
                 throw new Exception(errorString("Locale"));
         }
 
-        public bool Validate(LibraryBook libraryBook) => !libraryBook.Book.Audio_Exists;
+        public override bool Validate(LibraryBook libraryBook) => !libraryBook.Book.Audio_Exists;
```

Comment:
Is there a reason validate and Validate can't be combined?

Reply:
Different workflow logic. The private one should prob be renamed.

Public Validate is its Processable workflow: 'is this book a candidate for download/decrypt? if already liberated, then: no'.

Private validate: We know the book is a candidate for liberation. Do internal validation about the book's metadata.

If the public method also processed the private checks, one of these would be true for invalid metadata:

* return false. Indistinguishable from a book which has already been liberated. Bad metadata is hidden.
* throw. This implies there was an error seeing if this book is candidate for liberation. Possible, but it feels like it overloads what the public Validate is intended to check